### ISSUE TITLE
[Android] AIR app selected as launcher show black screen on home button press

### ISSUE THREAD

**Issue by itlancer** [airsdk/Adobe-Runtime-Support#1194]
## Problem Description
AIR app selected as Android launcher show black screen on home button press.
It blocks application usage at all.

It has been tested with multiple AIR versions (even with latest AIR 33.1.1.620) with multiple Android devices with different OS versions with different AIR applications with different architectures.
Same problem in all cases almost every time.

Related issues (not the same):
https://github.com/airsdk/Adobe-Runtime-Support/issues/1003
https://github.com/airsdk/Adobe-Runtime-Support/issues/1002
https://github.com/airsdk/Adobe-Runtime-Support/issues/698
https://github.com/airsdk/Adobe-Runtime-Support/issues/6

## Steps to Reproduce
1) Package AIR application (even "empty") for Android with ability to be as launcher (set up with manifest).
2) Install application to any Android device.
3) Launch application. You will see red stage color in attached example.
4) Press home button.

Application example with sources and application XML attached.
[android_launcher_black_screen_bug.zip](https://github.com/airsdk/Adobe-Runtime-Support/files/7228568/android_launcher_black_screen_bug.zip)

AIR application code:
```
package {
	import flash.display.Sprite;
	import flash.events.Event;
	
	public class AndroidLauncherBlackScreenBug extends Sprite {
		
		public function AndroidLauncherBlackScreenBug() {
			addEventListener(Event.ADDED_TO_STAGE, addedToStage);
		}
		
		private function addedToStage(e:Event):void {
			stage.color = 0xff0000;
			trace("addedToStage");
		}
	}
}
```

Application XML:
```
<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
<application xmlns="http://ns.adobe.com/air/application/33.1">
    <id>android.launcher.black.screen.bug</id>
    <versionNumber>1.0</versionNumber>
    <filename>android_launcher_black_screen_bug</filename>
    <description/>
    <name>android_launcher_black_screen_bug</name>
    <copyright/>
    <initialWindow>
        <content>android_launcher_black_screen_bug.swf</content>
        <systemChrome>standard</systemChrome>
        <transparent>false</transparent>
        <visible>true</visible>
        <fullScreen>true</fullScreen>
        <renderMode>direct</renderMode>
        <autoOrients>true</autoOrients>
        <maximizable>true</maximizable>
        <minimizable>true</minimizable>
        <resizable>true</resizable>
	<requestedDisplayResolution>high</requestedDisplayResolution>
    </initialWindow>
    <icon/>
    <customUpdateUI>false</customUpdateUI>
    <allowBrowserInvocation>false</allowBrowserInvocation>
    <android>
        <containsVideo>true</containsVideo>
		  <manifestAdditions><![CDATA[<manifest android:installLocation="auto">
		uses-sdk android:targetSdkVersion="30" />
		
		<application android:hardwareAccelerated="true">
			<meta-data android:name="android.max_aspect" android:value="4.0" />
			<activity>
				<intent-filter>
					<action android:name="android.intent.action.MAIN" />
					<category android:name="android.intent.category.LAUNCHER" />
					<category android:name="android.intent.category.HOME" />
					<category android:name="android.intent.category.LEANBACK_LAUNCHER" />
					<category android:name="android.intent.category.DEFAULT" />
				</intent-filter>
			</activity>
			
	</application>
</manifest>]]></manifestAdditions>
        <disableMediaCodec>true</disableMediaCodec>
    </android>
    <versionLabel></versionLabel>
</application>
```

**Actual Result:**
Application goes to black screen.
Video demonstration:

https://user-images.githubusercontent.com/10899066/134745505-798a3573-cbc4-41b5-9031-cbdf11bbbdc4.mp4

**Expected Result:**
Visually nothing happens cause application activity already launched.
Also `KeyboardEvent` will be dispatched with `Keyboard.HOME` `keyCode`.
And may be `InvokeEvent` will be dispatched if needed.

## Known Workarounds
none
Some time ago For some other Android devices worked but not 100% times (with AIR 33 now seems it doesn't work at all):
- Change Stage::quality https://stackoverflow.com/questions/19515570/air-for-android-black-screen-after-device-alarm
- Use "blackScreenHelper"
https://docs.airnativeextensions.com/docs/application/general-helpers/#usage
https://forum.starling-framework.org/d/4540-starling-turns-into-black-screen
- Make few Stage::setOrientation calls with different orientation

**Comment by itlancer** [comment]
Issue still exists with latest AIR 50.2.3.8 and can be reproduced using steps:
1) Set the launcher to a default normal one.
2) Uninstall the sample AIR application (if it being installed).
3) Install sample application.
4) Open sample application directly.
5) Pull down the top bar to go to Settings, and change the launcher to use this sample application.

**Comment by itlancer** [comment]
Fixed with latest AIR 50.2.4.1.
Thanks!

But there are still exists related issues:
https://github.com/airsdk/Adobe-Runtime-Support/issues/2396
https://github.com/airsdk/Adobe-Runtime-Support/issues/2017
https://github.com/airsdk/Adobe-Runtime-Support/issues/151
